week 3-4/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>To-do App </title>
+  </head>
+  <body>
+   <div class="main-div">
+        <input type="text" id="input"
```

Comment:
You did not close this input tag, thus not making it possible for the Todo list to work